uint32 preamble
uint8 mode
float32 sta_ref
float32 str_ref
float32 yaw_ref
float32 yaw_rate_ref
float32 swa
float32 swar
float32 reserved0
float32 reserved1
float32 reserved2
float32 reserved3
float32 speed_ref
float32 acc_ref
float32 jerk_ref
float32 gpa_ref
float32 ebrake_ref
float32 hbrake_ref
float32 reserved4
float32 reserved5
float32 reserved6
uint8 gear
uint8 mode_fb
uint8 switch_fb
float32 steer_fb
float32 acc_fb
float32 ebrake_fb
uint8 hbrake_fb